<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <node pkg="tf" type="static_transform_publisher" name="world_to_cam"
          args="0 0 -2.0 0 1.57 0 world smarteye_odom 10" />
  <!-- Run driver type:param:str|int|double|bool|yaml -->
  <node name="smarteye_with_viewpoint" pkg="smarteye_embedded_ros" type="smarteye_with_viewpoint" output="screen">
  <param name="eye2hand_config_addr" type="string" value="$(find smarteye_embedded_ros)/config/camera_aubo_config.yaml" />
  <param name="open_camera_flag" type="int" value="0"/>
  <param name="oblique_take_picture_the_max_height" type="double" value="1.359"/>
  <param name="smarteye_frame_id" type="string" value="smarteye_odom"/>
  <param name="diffrential_num" type="int" value="5"/>
  <param name="save_pcd_name" type="string" value="/data/ros/pcd_test.pcd"/>
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find smarteye_embedded_ros)/rviz/smart_eye.rviz"/>

</launch>

